scripts/preloadHTTPSE.js

```diff
@@ -33,84 +32,40 @@ const downloadRulesets = (dir, cb) => {
 }
 
 const buildDataFiles = () => {
-  // Manually exclude sites that are broken until they are fixed in the next
-  // HTTPS Everywhere release.
-  const exclusions = {
-    'Nike.com.xml': 'breaks nikeplus.com',
-    'PJ_Media.xml': 'mixed content on https://pjmedia.com/instapundit/',
-    'Slashdot.xml': 'redirect loop on mobile slashdot.org',
-    'Delta.com.xml': 'https://delta.com does not redirect to https://www.delta.com',
-    'Cargo.xml': 'breaks cargocollective.com',
-    'TMZ.com.xml': 'breaks www.tmz.com',
-    'BusinessInsider.xml': 'breaks http://www.businessinsider.com/silicon-valley-100-2016-6?op=0',
-    'Tesco.xml': 'breaks tesco.com due to CSP mismatch',
-    'Vodafone.ie.xml': 'breaks pagination on http://shop.vodafone.ie/shop/phonesAndPlans/phonesAndPlansHome.jsp?subPage=phones&planFilter=onAccount',
-    'IDownloadBlog.xml': 'breaks http://www.idownloadblog.com/',
-    'EBay_static.com.xml': 'breaks suggested product image previews',
-    'Cisco.xml': 'breaks http://www.cisco.com/c/m/en_us/training-events/events-webinars/techwise-tv/listings.html',
-    'GQ.xml': 'mixed content on gq.com',
-    'Where_2_Get_It.xml': 'maps missing on http://us.coopertire.com/Customer-Care/Dealer-Locator.aspx?form=locator_search&addressline=92346',
-    'Thompson_Hotels.com.xml': 'missing stylesheets on http://www.thompsonhotels.com/'
-  }
-
-  const rulesets = JSON.parse(fs.readFileSync('./https-everywhere/chrome/content/rulesets.json', 'utf8'))
-
-  // Convert XML rules to JSON
-  for (let id in rulesets.rulesetStrings) {
-    let contents = rulesets.rulesetStrings[id]
-    parseString(contents, function (err, result) {
-      if (err) {
-        throw new Error('FATAL: error parsing XML: ' + contents)
-      }
-      // Exclude broken rules
-      const ruleset = result.ruleset
-      if (ruleset.$.f in exclusions) {
-        console.log('NOTE: Excluding rule', JSON.stringify(result))
-        ruleset.$.default_off = exclusions[ruleset.$.f]
-      }
-      rulesets.rulesetStrings[id] = result
-    })
-  }
-
-  console.log('Writing httpse.json')
-  fs.writeFileSync('./out/httpse.json', JSON.stringify(rulesets), 'utf8')
```

Comment:
we still need to create `httpse.json` since it's used on the Brave desktop browser (only mobile uses leveldb)